packages/gatsby-theme-newrelic/src/components/GlobalFooter.js

```diff
@@ -99,6 +104,7 @@ const GlobalFooter = ({ fileRelativePath, className, pageTitle }) => {
               fileRelativePath={fileRelativePath}
               variant={Button.VARIANT.OUTLINE}
               size={Button.SIZE.SMALL}
+              labels={issueLabels}
```

Comment:
We'll need the labels prop to be in the `CreateIssueButton` component above, not this `EditPageButton`